README.md

```diff
@@ -9,11 +9,37 @@ standalone tracing and observability platform, Traceable created this open sourc
 project for the software development community to use in their applications.
 
 ---
+## Quick-start
+If you want to see Hypertrace in action, you can quickly start Hypertrace via Docker.
 
+```
+git clone https://github.com/hypertrace/hypertrace.git
+cd hypertrace
+docker-compose -f docker/docker-compose.yml up
+```
+This will start all services required for Hypertrace. Once you see the service `Hypertrace-UI` start, you can visit Hypertrace UI at http://localhost:2020 . 
 
-### To start using Hypertrace
-- [Join the Hypertrace Workspace](https://www.hypertrace.org/get-started) on Slack to chat with other Hypertrace users.
-- You will be invited to a private channel where you can get the installer zip file along with instructions to install!
+If you have application instrumented to send traces to Zipkin or Jaeger, you are already covered with Hypertrace. Even if not you can run sample application using `docker-compose -f docker/docker-compose-zipkin-example.yml` or you can try Hypertrace with sample application by running
+
+```
+docker-compose -f docker/docker-compose.yml -f docker/docker-compose-zipkin-example.yml up
+```
+Example app will be served at http://localhost:8081 . You can visit app to generate some sample requests!
+
+## Deploying with Kubernetes
+Please refer to [deployments](https://docs.hypertrace.org/deployments/) section in documentation which lists down steps for deploying Hypertrace on different Kubernetes flavors across different operating systems along with all major cloud providers. You can find the helm charts and installation script with more details [here](/kubernetes).
+
+## Community
+[Join the Hypertrace Workspace](https://www.hypertrace.org/get-started) on Slack to connect with other users, contributors and people behind Hypertrace. 
+
+## Documentation
+Check out [Hypertrace documentation](https://docs.hypertrace.org) to know more about Hypetrace features, architecture and more cool insights!
+- [Getting started](https://docs.hypertrace.org/getting-started/)
```

Comment:
I don't think the bullet points are needed. just the doc link section, especially as this one basically says what's already here. The one below on deployments also was mentioned above. Anyway, less bullets means less links to drift.